gammapy/data/tests/test_utils.py

```diff
@@ -0,0 +1,36 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+from gammapy.data.utils import get_irfs_features
+from gammapy.utils.testing import (
+    assert_allclose,
+    requires_data,
+)
+from gammapy.data import DataStore
+
+
+@requires_data()
+def test_irfs_features():
+    selection = dict(
+        type="sky_circle",
+        frame="icrs",
+        lon="329.716 deg",
+        lat="-30.225 deg",
+        radius="2 deg",
+    )
+
+    data_store = DataStore.from_dir("$GAMMAPY_DATA/hess-dl3-dr1/")
+    obs_table = data_store.obs_table.select_observations(selection)
+    obs = data_store.get_observations(obs_table["OBS_ID"][:1])
+
+    position = SkyCoord(329.716 * u.deg, -30.225 * u.deg, frame="icrs")
+    names = ["edisp-bias", "edisp-res", "psf-radius"]
+    features_irfs = get_irfs_features(
+        obs,
+        energy_true="1 TeV",
+        position=position,
+        names=names,
+    )
+    assert_allclose(features_irfs[0]["edisp-bias"], 0.11587, rtol=1.0e-4)
+    assert_allclose(features_irfs[0]["edisp-res"], 0.36834, rtol=1.0e-4)
+    assert_allclose(features_irfs[0]["psf-radius"], 0.14149, rtol=1.0e-4)
```

Comment:
This seems unrelated to time filtering. Please move to a separate PR.

Reply:
Indeed, it allows to add some missing time coverage

Reply:
If this test add missing time coverage it means that unit tests for time are missing because `get_irfs_features` doesn't do time filtering so if we modify/remove this test for other reason we will loose coverage on time related functionnalities.

Reply:
do you improve the coverage simply form these 3 lines using HESS data where gtis are defined ?

```
    data_store = DataStore.from_dir("$GAMMAPY_DATA/hess-dl3-dr1/")
    obs_table = data_store.obs_table.select_observations(selection)
    obs = data_store.get_observations(obs_table["OBS_ID"][:1])
```

Reply:
OK, I remove it... Does not matter